LOCAL_ENVIRONMENT_SETUP.md

```diff
@@ -19,25 +21,17 @@ _Now, let’s proceed with the installation of Git._
 2. Please wait while the installation completes.
 3. Excellent, Git is now installed.
 
-### Install Node Version Manager
-
-_Now, let's move on to the next dependency: Node.js._
-
-_There are two valid options for installing Node.js on your system. You can either use an application called [Node Version Manager for Windows (NVM)](https://github.com/coreybutler/nvm-windows), which we will demonstrate in this tutorial, or directly download Node.js from the official website, especially if you’re using Linux or macOS. Both methods are acceptable, so you can choose the one that suits you best._
+### Install Node.js
 
-_The link to download NVM is on the screen, and you can also find it in the video description below or in the text version of this tutorial._
+_There are two valid options for installing Node.js on your system. You can either use an application called [Node Version Manager for Windows (NVM)](https://github.com/coreybutler/nvm-windows), or directly download ([https://nodejs.org/en](https://nodejs.org/en/download) Node.js from the official website, especially if you’re using Linux or macOS. Both methods are acceptable, so you can choose the one that suits you best._
```

Comment:
Hey @H1kirro, thank you very much for your help in improving this document. Just one small comment: would it be possible to fix how the NodeJS download link is displayed here? Thanks,